modules/ballerina-http/src/main/java/org/ballerinalang/net/http/CorsHeaderGenerator.java

```diff
@@ -108,22 +108,26 @@ public static void process(HTTPCarbonMessage requestMsg, HTTPCarbonMessage respo
         //6.2.3 - request must have access-control-request-method, must be single-valued
         List<String> requestMethods = getHeaderValues(Constants.AC_REQUEST_METHOD, cMsg);
         if (requestMethods == null || requestMethods.size() != 1) {
-            bLog.info(action + "not allowed request methods");
+            String error = requestMethods == null ? "Access-Control-Request-Method header is unavailable" :
+                    "multiple request methods";
```

Comment:
"multiple request methods" is it informative enough ?

Reply:
Changed